tests/Unit/CssInlinerTest.php

```diff
@@ -196,224 +86,6 @@ public function constructorWithNoHtmlDataThrowsException($html)
         new CssInliner($html);
```

Comment:
`AbstractHtmlProcessorTest` has a `constructorWithNoHtmlDataThrowsException` test so is this also a duplicate that can be removed?

Reply:
Done.